Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -1595,36 +1723,36 @@ private void OnDestroySessionCompleteCallback(ref DestroySessionCallbackInfo dat
                 return;
             }
 
-            // Before removing the session from our local data, we need to inform the owner of the session that we've left the session, if we're not the owner
-            // TODO: Validate that this gets to the members/owners of the session in time, and that we haven't already deleted the local information needed to get session information
+            // Before removing the Session from our local data, we need to inform the Owner of the Session that we've left the Session, if we're not the Owner
+            // TODO: Validate that this gets to the members/owners of the Session in time, and that we haven't already deleted the local information needed to get Session information
```

Comment:
"...and that the local information needed to get Session information has not already been deleted." is a version that removes the use of the word, "we."